appserver/tests/payara-samples/samples/pom.xml

```diff
@@ -15,20 +15,26 @@
     <name>Payara Samples - Profiled Tests</name>
 
     <modules>
+        <!-- This module must run first, because it collides with some change made by other test(s) -->
+        <module>payara-expression-config-properties</module>
```

Comment:
It shouldn't do? Tests should clean up after themselves.

Reply:
They reuse the managed/remote domain, so you can never get to the state before all tests. But yes, it seems some (or all?) tests leave some garbage behind them. Btw this one too.